### PR TITLE
Move ontologies to an ontologies folder

### LINKED ISSUE #904: Put ontologies in an ontologies folder
The gist root directory is mostly nicely partitioned, and I propose that we make it one step cleaner by putting the ontologies (there are 5 so far and others may be coming) into their own `ontologies` directory. In my client projects I typically go one level deeper, using `models/ontologies` alongside `models/taxonomies` and whatever we want to call controlled datasets that aren't taxonomies. We already have one of the latter in `gistMediaTypes.ttl`. 

Is it also possible to move `bundle.yaml` to `tools`? Not sure if this would have any adverse affects when running `onto_tool`.

### LINKED ISSUE #962: Update bundle.yaml to use onto_tool excludes option
The new `excludes` onto_tool option allows excluding files from an action rather than individually listing files to be included. See https://github.com/semanticarts/ontology-toolkit/issues/111, #922.

We can now replace

```
  includes:
    - gistCore.ttl
    - gistMediaTypes.ttl
    - gistPrefixDeclarations.ttl
    - gistSubClassAssertions.ttl
  ```
with
```
  includes:
    - '*.ttl'
  excludes:
    - gistValidationAnnotations.ttl
```

### PR DESCRIPTION
**Pull request by rjyounes**
Closes #904
Closes #962

### CODE REVIEW

**Comment by rjyounes** [comment]
@Jamie-SA Is this going to require any changes to the upload process or import IRI?

File: docs/release_notes/issue904-ontology-folder [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Minor Updates
+
+- Move ontology files into a root-level `ontologies` folder. [Issue #904](https://github.com/semanticarts/gist/issues/904).
```
Comment (rjyounes) [inline docs/release_notes/issue904-ontology-folder:3]:
```suggestion
- Moved ontology files into a root-level `ontologies` folder. [Issue #904](https://github.com/semanticarts/gist/issues/904).
```

File: bundle.yaml [hunk 2]
```diff
@@ -85,32 +85,33 @@ actions:
...
 - action: "sparql"
   message: "Generating rdfs:label and rdfs:comment for backward compatibility."
-  source: "{output}"
-  target: "{output}/rdfsAnnotations.ttl"
+  source: "{output}/ontologies"
+  target: "{output}/ontologies/rdfsAnnotations.ttl"
   format: "turtle"
   includes:
-    - "*.ttl"
+    - '*.ttl'
```
Comment (rjyounes) [inline bundle.yaml:112]:
```suggestion
    - "*.ttl"
```

File: bundle.yaml [hunk 3]
```diff
@@ -85,32 +85,33 @@ actions:
...
+    - gistValidationAnnotations.ttl
 - action: "definedBy"
   message: "Adding rdfs:isDefinedBy."
-  source: "{output}"
-  target: "{output}"
+  source: "{output}/ontologies"
+  target: "{output}/ontologies"
   includes:
-    - "*.ttl"
+    - '*.ttl'
```
Comment (rjyounes) [inline bundle.yaml:105]:
```suggestion
    - "*.ttl"
```

File: bundle.yaml [hunk 4]
```diff
@@ -85,32 +85,33 @@ actions:
...
     to: "{version}"
   rename:
     from: "(.*)\\.ttl"
     to: "\\g<1>{version}.ttl"
   includes:
-    - gistCore.ttl
-    - gistMediaTypes.ttl
-    - gistPrefixDeclarations.ttl
-    - gistSubClassAssertions.ttl
+    - '*.ttl'
```
Comment (stevenchalem) [inline bundle.yaml]: @rjyounes Does the exclude override the include here?

Comment (rjyounes) [reply bundle.yaml]: yes

**Review by stevenchalem — APPROVED** [review]

**Pull request merged**